src/components/Articles.jsx

```diff
@@ -1,15 +1,51 @@
-import React from "react";
```

Comment:
`Articles` and `Articlelist` are both incredible similar names for components - what are they? Can you differentiate their name more? 

It's potentially easier to move the map on line 35 of Articlelist into Articles.jsx and then make Articlelist into a 'ArticleCard' component to keep lines 37 - 43 out of Article.jsx. This would ease the naming convention into something more readable. But totally your choice.